lib/rails_values/email_address.rb

```diff
@@ -14,8 +14,7 @@ def initialize(address)
       freeze
     end
 
-    delegate :local, :display_name, :format, to: :mail_address
-
+    delegate :local, :display_name, to: :mail_address
```

Comment:
`format` on mail address originally called as part of `EmailAddress` to support display names, but I think it really only belongs in subclass now